Sources/UI/Web View/WebViewController.swift

```diff
@@ -136,6 +148,15 @@ extension UIViewController {
         let webViewController = WebViewController()
         webViewController.url = url
         webViewController.title = title
-        present(UINavigationController(rootViewController: webViewController), animated: animated)
+        present(WebViewNavigationController(rootViewController: webViewController), animated: animated)
+    }
+}
+
+fileprivate class WebViewNavigationController: UINavigationController {
+    override func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        if let webVC = viewControllers.filter({ $0 is WebViewController }).first as? WebViewController {
```

Comment:
I don't like `VC` as a shortcut for `ViewController`. You won't find it in our or Apple SDK.